gbtis/Kinect.cs

```diff
@@ -127,6 +141,26 @@ public class Kinect {
         }
 
         /// <summary>
+        /// Calculates the average finger position over the last
+        /// POINT_SAMPLE_SIZE frames
+        /// </summary>
+        /// <param name="x">The latest x coordinate</param>
+        /// <param name="y">The latest y coordinate</param>
+        /// <returns>The average position</returns>
+        private Point getAverageFingerTipPosition(float x, float y) {

```

Comment:
Will this not be too slow at 30fps?

Why not do something like:
  
    rollingAverage (double avg, double new_sample) {

    avg -= avg / N;
    avg += new_sample / N;

    return avg;
}